models/turbine_models/custom_models/sd_inference/utils.py

```diff
@@ -1,96 +1,198 @@
 import iree.compiler as ireec
 import numpy as np
+import os
 import safetensors
 import re
 from diffusers import (
     PNDMScheduler,
+    EulerDiscreteScheduler,
 )
 
+winograd_params = "keys=unet.down_blocks.2.resnets.0.conv2.weight keys=unet.down_blocks.2.resnets.1.conv1.weight keys=unet.down_blocks.2.resnets.1.conv2.weight keys=unet.mid_block.resnets.0.conv1.weight keys=unet.mid_block.resnets.0.conv2.weight keys=unet.mid_block.resnets.1.conv1.weight keys=unet.mid_block.resnets.1.conv2.weight keys=unet.up_blocks.0.resnets.0.conv2.weight keys=unet.up_blocks.0.resnets.1.conv2.weight keys=unet.up_blocks.0.resnets.2.conv2.weight keys=unet.up_blocks.0.resnets.0.conv1.weight keys=unet.up_blocks.0.resnets.1.conv1.weight keys=unet.up_blocks.0.resnets.2.conv1.weight keys=unet.up_blocks.0.upsamplers.0.conv.weight"
```

Comment:
We probably shouldn't have all of these winograd_params and flags hardcoded. Probably fine for now but might want a todo to address this